src/utils/checkToken.ts

```diff
@@ -0,0 +1,14 @@
+import jwt from 'jsonwebtoken';
+
+require('dotenv').config();
+
+const checkToken = (jwtToken: string): string | undefined => {
+  const jwtSecret: string | undefined = process.env.JWT_SECRET;
+  if (jwtSecret !== undefined) {
+    const userId: any = jwt.verify(jwtToken, jwtSecret);
+    // eslint-disable-next-line no-underscore-dangle
+    return userId._id;
+  }
+  return undefined;
```

Comment:
```suggestion
  const jwtSecret: string = process.env.JWT_SECRET as string;
  const userId: any = jwt.verify(jwtToken, jwtSecret);
  // eslint-disable-next-line no-underscore-dangle
  return userId._id;
```

process.env 가 없는 경우는 서버가 구동하는 도중에 처리가 필요한 예외는 아닌 것 같아서, 이런 식으로 바꾸어서 lint 에러를 피해가는 것이 더 간결할 수 있을 것 같네요 ☺